packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -676,8 +676,8 @@ export class RichText extends Component {
 	forceSelectionUpdate( start, end ) {
 		if ( ! this.needsSelectionUpdate ) {
 			this.needsSelectionUpdate = true;
-			this.selectionStart = start;
-			this.selectionEnd = end;
+			this.selectionStart = isNaN( start ) ? 0 : start;
```

Comment:
I think we can fix this particular crash without these changes. On the other hand, there could be other untested code paths that lead to here with an invalid state, so I don't think it hurts to leave the isNaN check here.